src/convertToObject.js

```diff
@@ -5,8 +5,18 @@
  *
  * @return {object}
  */
-function convertToObject(sourceString) {
-  // write your code here
+
+function convertToObject(stylesString) {
+  const styleObject = {};
+  const declarations = stylesString.split(';').filter(Boolean);
```

Comment:
interesting move, but this only filters out just empty strings (' ' will pass).
If you start filtering and validating, it makes sense to check, that each substring has one `:` symbol inside that go not on the first or last position